src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,13 @@ public class User {
     private String password;
     private Integer age;
 
+    public User(Long id, String login, String password, Integer age) {
+        this.id = id;
```

Comment:
Do not set Id through the constructor. There are cases when user is not saved in db and does not have any id